scripts/shared/clusters.sh

```diff
@@ -2,15 +2,15 @@
 
 ## Kubernetes version mapping, as supported by kind ##
 # See the release notes of the kind version in use
-DEFAULT_K8S_VERSION=1.23
+DEFAULT_K8S_VERSION=1.24
```

Comment:
This change seems to be part of the following PR - https://github.com/submariner-io/shipyard/pull/859/files